README.md

```diff
@@ -1,88 +1,70 @@
 # stylelint-order [![Build Status][ci-img]][ci] [![npm version][npm-version-img]][npm] [![npm downloads last month][npm-downloads-img]][npm]
 
-A plugin pack of order related linting rules for [stylelint]. Every rule support autofixing (`stylelint --fix`).
+A plugin pack of order-related linting rules for [stylelint]. Every rule support autofixing (`stylelint --fix`).
 
 ## Installation
 
-First, install [stylelint]:
+Install [stylelint] & `stylelint-oder`:
 
 ```
-npm install stylelint --save-dev
-```
-
-Then install plugin:
-
-```
-npm install stylelint-order --save-dev
+npm install stylelint stylelint-order --save-dev
 ```
 
 ## Usage
 
-Add `stylelint-order` to your stylelint config plugins array, then add rules you need to the rules list. All rules from stylelint-order need to be namespaced with `order`.
+Add `stylelint-order` to your stylelint config `plugins` array, then add rules you need to the rules list. All rules from stylelint-order need to be namespaced with `order`.
 
-Like so:
+### `.stylelintrc` Example
 
-```js
-// .stylelintrc
+```json
 {
-	"plugins": [
-		"stylelint-order"
-	],
-	"rules": {
-		// ...
-		"order/order": [
-			"custom-properties",
-			"declarations"
-		],
-		"order/properties-alphabetical-order": true
-		// ...
-	}
+  "plugins": ["stylelint-order"],
+  "rules": {
+    "order/order": ["custom-properties", "declarations"],
+    "order/properties-alphabetical-order": true
+  }
```

Comment:
Let's remove heading and reformat JSON to be more structured and use tabs:

<pre><code>```json
{
	"plugins": [
		"stylelint-order"
	],
	"rules": {
		"order/order": [
			"custom-properties",
			"declarations"
		],
		"order/properties-alphabetical-order": true
	}
}
```</code></pre>